contracts/utils/Utils.sol

```diff
@@ -45,11 +42,11 @@ library Utils {
         }
     }
 
-    function _getPositionHash(bytes32[] memory actionHashes) internal pure returns (bytes32) {
-        return keccak256(abi.encode(actionHashes));
+    function _getPositionHash(bytes32[] memory actionHashes) internal view returns (bytes32) {
```

Comment:
try and keep functions pure unless necessary

Reply:
Why? 
It is necessary here since we're using `address(this)`

Now could we do without `address(this)` and do that bit of complexity in The Graph?
What's the tradeoff? This doesn't cost gas either.